core/server/api_container/server/api_container_service.go

```diff
@@ -137,14 +140,17 @@ func (apicService *ApiContainerService) RunStarlarkScript(args *kurtosis_core_rp
 	experimentalFeatures := args.GetExperimentalFeatures()
 	cloudUserId := shared_utils.GetOrDefaultString(args.CloudUserId, defaultCloudUserId)
 	cloudInstanceID := shared_utils.GetOrDefaultString(args.CloudInstanceId, defaultCloudInstanceId)
+	ApiDownloadMode := shared_utils.GetOrDefault(args.ImageDownloadMode, defaultImageDownloadMode)
+
+	downloadMode := image_download_mode.FromAPI(ApiDownloadMode)
```

Comment:
ahh - right idea here, we need some way to translate the representation of `imageDownloadMode` `kurtosis_core_rpc_api_bindings` to the representation in `container_engine_lib`. 

However, I propose to do this, we use an existing pattern in the codebase. 

First, let's change the representation of `image download` mode in `container-engine-lib` to how it's done for in `[service status](https://github.com/kurtosis-tech/kurtosis/blob/main/container-engine-lib/lib/backend_interface/objects/service/service_status.go)` 

Then, let's do the translation of `kurtosis_core_rpc_api_bindings.ImageDownloadMode` to `image_download_mode.ImageDownloadMode` to look like how it's done for service status [here](https://github.com/kurtosis-tech/kurtosis/blob/563798c82954d807d926d8e92d67bc39c1b832eb/core/server/api_container/server/api_container_service.go#L944).

All this is doing is taking the existing `switch` statement/translation you have in the `container-engine-lib` and moving it to this file in `apic`, but this pattern is preferred bc it removes the dependency of `container-engine-lib` on `apic`(`container-engine-lib` doesn't need to know about how `apic` is representing `ImageDownloadMode`, only `apic` needs to know about that)